lnst/RecipeCommon/Ping/Recipe.py

```diff
@@ -73,7 +73,9 @@ def ping_test(self, ping_configs):
 
         for _, pingjob in ping_array:
             try:
-                pingjob.wait()
+                pingjob.wait(
+                    timeout=self._estimate_ping_duration(pingconf),
```

Comment:
I missed that this is a single ping, so the estimate should be per ping, not the whole set of pings.

Reply:
Doh, no, no, no. It's correct. The pingconf is wrapping multiple pings, so this is correct approach.